src/web/mjs/engine/Connectors.mjs

```diff
@@ -24,16 +25,22 @@ export default class Connectors {
         ];
         let userPlugins = await this._loadPlugins('hakuneko://plugins/');
         let internalPlugins = await this._loadPlugins('hakuneko://cache/mjs/connectors/');
+        let favoritePlugins = Engine.Storage.downloadedConnectors;
 
         await this.register(systemPlugins);
         await this.register(userPlugins);
         await this.register(internalPlugins);
+        await this.registerfav(favoritePlugins);
```

Comment:
Is this required?
All websites should be registered by `await this.register(userPlugins);` and `await this.register(internalPlugins);`. If there is a website in favorites, that is not in `internalPlugins` or `userPlugins` that software is doing something wrong.